soroban-sdk/src/address.rs

```diff
@@ -388,9 +388,9 @@ impl crate::testutils::Address for Address {
     }
 }
 
-#[cfg(not(target_family = "wasm"))]
+#[cfg(any(not(target_family = "wasm"), test, feature = "testutils"))]
```

Comment:
Why did this line need changing? I don't remember the full backstory but the way fns are behind cfg is split, there's stuff behind it for testutils, and all the scval stuff is behind the non-wasm target.